.github/workflows/aot-check.yml

```diff
@@ -22,6 +22,11 @@ jobs:
       with:
         fetch-depth: 1
 
+    - name: Setup .NET 9.0.x
+      uses: actions/setup-dotnet@v4.0.1
+      with:
+        dotnet-version: 9.0.x
+
     - name: Runs powershell script
       id: aot-powershell
       run: build\test-aot.ps1 'net8.0'
```

Comment:
Target `net9.0`?
```suggestion
      run: build\test-aot.ps1 'net9.0'
```

Reply:
I was trying to keep this the same and still target net8 here.